app/src/components/governance/ProposalHeader.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <header class="header">
+    <div class="header-top">
+      <div>
+        <button><i class="material-icons">arrow_back</i>Proposals</button>
+      </div>
+      <div>
+        <span v-if="votingPeriod">Voting Period</span>
+        <span v-if="depositPeriod">Deposit Period</span>
+      </div>
+      <div>
+        <button><i class="material-icons">link</i>Link</button>
+        <button><i class="material-icons">share</i>Twitter</button>
+        <button>Vote</button>
+      </div>
+    </div>
+    <h2>{{ title }}</h2>
+    <div>
+      <div>
+        Proposer:
+        {{ proposer.img }}
+        {{ proposer.name }}
+        {{ proposer.address }}
+      </div>
+      <p>{{ summary }}</p>
+    </div>
+    <nav>
+      <ul class="supporting-links">
+        <li><a href="">Validators</a></li>
```

Comment:
what are those links?

Reply:
they will scroll to the section on the page

Reply:
Alright. Then the class "supporting-links" is confusing

Reply:
yep - not the right class name.